<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitManager</class>
 <widget class="QDialog" name="GitManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Backup and History Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QListWidget" name="listHistory">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelHistory">
       <property name="text">
        <string>History</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnSaveBackup">
         <property name="text">
          <string>Save backup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRestoreBackup">
         <property name="text">
          <string>Restore backup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRevertStep">
         <property name="text">
          <string>Revert step</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editRevert"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelCommit">
       <property name="text">
        <string>Commit:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelOutput">
     <property name="text">
      <string>History manager output</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="editOutput">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
